packages/node/src/lib/events/events.interface.ts

```diff
@@ -49,6 +49,7 @@ export type ITriggerOverrides = {
     | 'plivo'
     | 'postmark'
     | 'sendgrid'
+    | 'chat'
```

Comment:
Can you please elaborate on this change, this name seems too generic

Reply:
It’s leftovers from other PR I was working on, file reverted and line removed